src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -337,6 +366,10 @@ export default {
 		max-height: 64px;
 	}
 
+	.mimeicon {
+		min-height: 120px;
```

Comment:
`120px` or `128px`?

Reply:
I picked that value as it looked good. But since we're talking icons, maybe 128px is better suited. Will adjust.